### PR TITLE
Add latency checks to peer connection and status output

### PR DESCRIPTION
**Pull request by pascal-fischer**
## Describe your changes
This PR uses the ice agents healthchecks to extend out connection object with the latency between peers. This is also added to the status details output of the CLI.

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [x] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: client/cmd/status.go [hunk 1]
```diff
@@ -639,6 +659,11 @@ func parsePeers(peers peersStateOutput, rosenpassEnabled, rosenpassPermissive bo
 				"  Transfer status (received/sent) %s/%s\n"+
 				"  Quantum resistance: %s\n"+
 				"  Routes: %s\n",
+				"  Transfer status (received/sent): %s/%s\n"+
```
Comment (lixmal) [inline client/cmd/status.go]: Is this a dupe?

Comment (pascal-fischer) [reply client/cmd/status.go]: Ah yeah I think there went something wrong during merge. I will check the default status output

File: client/internal/peer/conn.go [hunk 2]
```diff
@@ -234,6 +237,17 @@ func (conn *Conn) reCreateAgent() error {
...
 
+	err = conn.agent.OnSuccessfulSelectedPairBindingResponse(func(p *ice.CandidatePair) {
+		err := conn.statusRecorder.UpdateLatency(conn.config.Key, p.Latency())
+		if err != nil {
+			log.Debugf("failed to update latency for peer %s: %s", conn.config.Key, err)
+			return
+		}
+	})
+	if err != nil {
+		return err
```
Comment (lixmal) [inline client/internal/peer/conn.go:248]: Can you annotate this error with some context? Makes debugging from the logs easier

**Review by lixmal — DISMISSED** [review]

File: client/internal/peer/conn.go (hunk 2 above)
Comment (lixmal) [reply client/internal/peer/conn.go:248]:
```suggestion
		return fmt.Errorf("select pair binding response: %w", err)
```

**Review by lixmal — APPROVED** [review]

**Pull request merged**